tensorflow/compiler/mlir/xla/experimental/conv_emitter/conv_emitter.cc

```diff
@@ -217,7 +219,7 @@ mlir::Operation* HoistAndFix(llvm::iplist<mlir::Operation>::iterator begin_op,
       op.moveBefore(&new_loops.back().getBody()->back());
     }
     CHECK_EQ(ancestors.size(), new_loops.size());
-    for (int i = 0; i < ancestors.size(); i++) {
+    for (decltype(ancestors)::size_type i = 0; i < ancestors.size(); i++) {
```

Comment:
This is as usual: you're bring back some unsigned semantics for the loop induction variable, I won't approve these changes.

Here it'd be better to avoid a raw loop anyway:

```
for (auto ancestor_and_new : llvm::zip(ancestors, new_loops) {
    replaceAllUsesInRegionWith(
      std::get<0>(ancestor_and_new).getInductionVar(),
      std::get<1>(ancestor_and_new).getInductionVar(),
      new_loops.back().region());
}
```